server/test/support.js

```diff
@@ -76,12 +76,26 @@ const seed = async () => {
   await document.publish();
   await collection.reload();
 
+  const attachment = await Attachment.create({
```

Comment:
Could we move this into the `factories` file – you'll see the pattern in there. This support method is somewhat deprecated as we're trying to be more specific about which test data is created to make the suite run as fast as possible 😄

Reply:
moved to `factories`. Think I followed the pattern correctly, it did make the test a bit more verbose but I assume that is the point.